airbyte-webapp/src/pages/routes.tsx

```diff
@@ -142,6 +144,15 @@ export const Routing = () => {
     }
   }, [workspace]);
 
+  const customer = {
+    external_id: workspace.customerId
+  };
+  Storytime.init({
```

Comment:
in case this is helpful (after our call) -- this is how we initialize Storytime in our dashboard in a `useEffect` hook: https://github.com/papercups-io/papercups/blob/master/assets/src/components/Dashboard.tsx#L105-L127

hopefully that improves things 🤞

Reply:
Thanks got it to work properly!